networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -12,7 +12,6 @@ IPsec is disabled by default. You can enable IPsec either during or after instal
 
 The following support limitations exist for IPsec on a {product-title} cluster:
 
-* You must disable IPsec before updating to {product-title} 4.15. There is a known issue that can cause interruptions in pod-to-pod communication if you update without disabling IPsec. (link:https://issues.redhat.com/browse/OCPBUGS-43323[*OCPBUGS-43323*])
```

Comment:
Removed as per this [Slack](https://redhat-internal.slack.com/archives/C08DNAFC85T/p1742459699316549) thread.